src/index.tsx

```diff
@@ -77,10 +78,12 @@ export const getResetPasswordTemplate = (
 
 export const getTransactionTemplate = (
   props: TransactionTemplateProps,
+  isResend?: boolean,
   options?: Mjml2HtmlOptions
 ) => {
   const intl = initIntl();
-  const subject = getLocalizedTransactionSubject(intl, props.amount);
+  let subject = getLocalizedTransactionSubject(intl, props.amount);
+  if (isResend) subject = getResendText(intl).concat(subject);
```

Comment:
```suggestion
let subject = intl.formatMessage({ id: 'transaction.subject' }, { amount: amount });
if (isResend) subject = intl.formatMessage({ id: 'resend' }).concat(subject);
```
Therefore, we can remove those getLocalizedXXXX, cleaner code

Reply:
true! Removed already in next commint